ppl/src/main/java/org/opensearch/sql/ppl/utils/ArgumentFactory.java

```diff
@@ -135,6 +136,17 @@
         .singletonList(new Argument("noOfResults", new Literal(10, DataType.INTEGER)));
   }
 
+  /**
+   * Get list of {@link Argument}.
+   *
+   * @param ctx KmeansCommandContext instance
+   * @return the list of arguments fetched from the kmeans command
+   */
+  public static List<Argument> getArgumentList(KmeansCommandContext ctx) {
+    return Collections
+            .singletonList(new Argument("k", getArgumentValue(ctx.k)));
```

Comment:
We can support other 2 parameters for Kmeans : `iterations` and `distanceType` , refer to this code https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/parameter/KMeansParams.java#L39-L43